Documentation/Books/Manual/Administration/ManagingUsers/AuthorizationModel.md

```diff
@@ -1,48 +1,22 @@
-
-| action                | collection | collection database | _system database |
-|-----------------------|------------|---------------------|------------------|
-| create a document     | rw         |                     |                  |
-| read a document       | r          |                     |                  |
-| update a document     | rw         |                     |                  |
-| drop a document       | rw         |                     |                  |
-| create a collection   |            | rw                  |                  |
-| drop a collection     |            | rw                  |                  |
-| truncate a collection | rw         |                     |                  |
-| create an index       |            | rw                  |                  |
-| drop an index         |            | rw                  |                  |
-| create a database     |            |                     | rw               |
-| drop a database       |            |                     | rw               |
-| create a user         |            |                     | rw               |
-| update a user         |            |                     | rw               |
-| drop a user           |            |                     | rw               |
-
-
 Internal representation
 -----------------------
 
-In general administrator will use the *@arangodb/users* module to create users
-and administrate their rights.
-
-These rights are stored in the *_users* table in the *_system* database.
-
-The new authorization model allows to add other types of rights.  Currently
-supported types are read and write permissions.  Every database object has at
-least a permissions attribute and collections attribute. The permissions
-sub-object holds the rights at dabase level while the collections sub-object
-holds an object with collection names. Every collection name object holds a
-permissions object which holds permissions for collection level authorization.
+In general administrator will use the *@arangodb/users* module to
+create users and administrate their rights. This is a description of
+the internal format used to describe users and their privileges.  This
+format is stored in the collection *_users* in the *_system* database.
+
+Note that this format is subject to change without notice, therefore
+you should never access this table directly. Always use the functions
+supplied by the *users* module.
+
+The new authorization model allows to add other types of rights.
+Currently supported types are read and write permissions.  Every
+database object has at least a permissions attribute and collections
+attribute. The permissions sub-object holds the rights at dabase level
+while the collections sub-object holds an object with collection
+names. Every collection name object holds a permissions object which
+holds permissions for collection level authorization.
```

Comment:
I'll love to see an example in this file with database '*' and especially a combination of database '*' with explicit & '*' collection.